src/cpr-tooltip.js

```diff
@@ -92,6 +92,9 @@ function updateTooltip() {
 	tooltipEl.style.borderRadius = '2px';
 	tooltipEl.style.boxShadow = '0 1px 4px 0 rgba(0, 0, 0, .26)';
 	tooltipEl.style.zIndex = '100000';
+	tooltipEl.style.maxWidth = '340px';
+	tooltipEl.style.maxHeight = '340px';
+	tooltipEl.style.textOverflow = 'ellipsis';
```

Comment:
Does this work right? Will you also need `overflow: hidden` and `whitespace: nowrap`? https://css-tricks.com/snippets/css/truncate-string-with-ellipsis/